README.md

```diff
@@ -19,6 +19,7 @@ The repository can be found on git at [libmraa](https://github.com/intel-iot-dev
 2. Build a docker image by using the following command
 ```
 sh> cd device-grove-c
+sh> make version 
 sh> docker build . -t < device-grove-c > -f ./scripts/Dockerfile.alpine-3.11
```

Comment:
should this line not read simply
`sh> docker build . -t device-grove-c -f ./scripts/Dockerfile.alpine-3.11`

Reply:
<> - This is to indicate the image name can be anything and not fixed to device-grove-c. I can remove if it is not clear.

Reply:
Yes please, maybe just say underneath "replacing `device-grove-c` with your preferred image name if necessary" or something